lib/sgtn-client/api/translation.rb

```diff
@@ -54,7 +63,10 @@ def self.getString_f(component, key, args, locale, *optionals)
          return s
       end
 
-      def self.getStrings(component, locale)
+      def self.getStrings(component, locale=SgtnClient::Config.configurations.default)
+        if locale.nil?
```

Comment:
Put this logic inside SgtnClient::Config.configurations so that you won't have to repeat it here and elsewhere.

Reply:
As the APIs like getString can be call separately, so seems not good to put the logic in SgtnClient::Config.configurations.